src/ivoc/idraw.cpp

```diff
@@ -152,7 +152,7 @@ void OcIdraw::text(Canvas*,
         out << font->name() << font->size() << "SetF\n";
     } else {
         out << "\
-%I f -*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*\n\
+%I f -*-fixed-medium-r-normal-*-12-*-*-*-*-*-*-*\n\
 Helvetica 12 SetF\n\
```

Comment:
does it matter that there's a `Helvetica` here?

Reply:
Good question, I was trying to find some documentation about this formatted file, but was not able...

Reply:
Several times in the past I made brief attempts to replace Helvetica with another proportional standard font but never reached a satisfactory result. In the old days, Helvetica was a standard postscript font.  InterViews has a complicated way of describing what font to use. Always in the past the fallback to fixed font resulted in very ugly text formatting in the InterViews windows. On linux the Helvetica font is installed with ```sudo apt install xfonts-100dpi```